packages/squiggle-lang/src/reducer/lambda.ts

```diff
@@ -155,33 +160,39 @@ export class BuiltinLambda extends Lambda {
   }
 
   parameterString() {
-    return this._definitions.map(fnDefinitionToString).join(" | ");
+    return this._signatures.map(fnDefinitionToString).join(" | ");
   }
 
-  definitions(): FRType<any>[][] {
-    return this._definitions.map((d) => d.inputs);
+  parameterCountString() {
+    return `[${this._signatures.map((d) => d.inputs.length).join(",")}]`;
```

Comment:
This needs to go through `parameterCounts`, otherwise the result has duplicates:

> Error: Calculator function needs [1,2,3,4,5,6,1,2,3,4,5] parameters, but 0 fields were provided.

https://squiggle-website-git-lambda-refactor-quantified-uncertainty.vercel.app/playground?v=dev#code=eNqrVkpJTUsszSlxzk9JVbJSck7MSS7NSSzJL9LLTcxO1aiOyVNQSMuzUsit0AEzM1NzUoqtFKJjY%2FJqNZVqAdenFeA%3D